lib/puppet.rb

```diff
@@ -36,6 +37,9 @@ module Puppet
   require 'puppet/environments'
 
   class << self
+    GettextSetup.initialize(File.absolute_path('../locales', File.dirname(__FILE__)))
```

Comment:
Naw, which is why I merged it. :) The reasoning is that before, we actually had scoping considerations (with `include`) but now we don't. Once that consideration has been dropped this can be located anywhere, basically. While something like this would generally be outside of a module/class definition because it's not related to the module/class definition, there's no reason to. I only commented to share this thought in case it's useful - but this is a very small detail and it's pretty bike sheddy.